.github/CODEOWNERS

```
@@ -15,6 +15,7 @@
 /examples/create-react-app               @TooTallNate @EndangeredMassa @trek @onsclom @jeffsee55 @erikareads @Timer
 /examples/nextjs                         @ijjk @ztanner @huozhi
 /packages/node                           @TooTallNate @EndangeredMassa @trek @onsclom @jeffsee55 @erikareads @Kikobeats
+/packages/related-projects               @TooTallNate @EndangeredMassa @trek @onsclom @jeffsee55 @erikareads @Kikobeats @tknickman @mknichel
```

```suggestion
/packages/related-projects               @tknickman @mknichel
```

I think this should be restricted to only you two. That way, everyone else isn't notified on every change to this package.